### ISSUE TITLE
MkNotification.java:43: Implement tests for...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1418]
The puzzle `1407-bd218f2f` from #1407 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/dfe7f7cf3d5bfc6318ff2b284e9f04c66afe32cd/src/main/java/com/jcabi/github/mock/MkNotification.java#L43-L43

The puzzle was created by George Aristy on 23-May-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1418](https://github.com/jcabi/jcabi-github/issues/1418) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1418](https://github.com/jcabi/jcabi-github/issues/1418) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1501563106); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)com.zerocracy.Par@4106833

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on review/merge of PR #1420

**Comment by 0crat** [comment]
> @0crat waiting on review/merge of PR #1420  ([here](https://github.com/jcabi/jcabi-github/issues/1418#issuecomment-395078502))

@llorllale The impediment for [#1418](https://github.com/jcabi/jcabi-github/issues/1418) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1407-bd218f2f` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1418](https://github.com/jcabi/jcabi-github/issues/1418) is now out of scope
